travis-scripts/fontforge.rb

```diff
@@ -87,43 +94,32 @@ def install
     args << "--without-giflib" if build.without? "giflib"
     args << "--without-libspiro" if build.without? "libspiro"
 
-    # Fix linker error; see: http://trac.macports.org/ticket/25012
+    # Fix linker error; see: https://trac.macports.org/ticket/25012
     ENV.append "LDFLAGS", "-lintl"
 
-    # And finding Homebrew's Python
-    ENV.append_path "PKG_CONFIG_PATH", "#{HOMEBREW_PREFIX}/Frameworks/Python.framework/Versions/2.7/lib/pkgconfig/"
-    ENV.prepend "LDFLAGS", "-L#{%x(python-config --prefix).chomp}/lib"
-
     # Reset ARCHFLAGS to match how we build
     ENV["ARCHFLAGS"] = "-arch #{MacOS.preferred_arch}"
 
+    # And for finding the correct Python, not always Homebrew's.
+    ENV.prepend "CFLAGS", "-I#{pydir}/include"
+    ENV.prepend "LDFLAGS", "-L#{pydir}/lib"
+    ENV.prepend_path "PKG_CONFIG_PATH", "#{pydir}/lib/pkgconfig"
+
     # Bootstrap in every build: https://github.com/fontforge/fontforge/issues/1806
     system "./bootstrap"
     system "./configure", *args
     system "make"
     system "make", "install"
+    # The name is case-sensitive. Don't downcase it when linking.
+    ln_s "#{share}/fontforge/osx/FontForge.app", prefix if build.with? "x11"
   end
 
   def post_install
-    # Link this to enable symlinking into /Applications with brew linkapps.
-    # The name is case-sensitive. It breaks without both F's capitalised.
-    # If you build with x11 now, it automatically creates an dynamic link from bin/fontforge
-    # to @executable_path/../Frameworks/Breakpad.framework/Versions/A/Breakpad which
-    # obviously doesn't exist given fontforge and FontForge.app are in different places.
-    # If this isn't fixed within a couple releases, consider dumping everything in libexec.
-    # https://github.com/fontforge/fontforge/issues/2022
-    if build.with? "x11"
-      ln_s "#{share}/fontforge/osx/FontForge.app", prefix
-      system "install_name_tool", "-change", "@executable_path/../Frameworks/Breakpad.framework/Versions/A/Breakpad",
-             "#{bin}/fontforge", "#{share}/fontforge/osx/FontForge.app/Contents/Frameworks/Breakpad.framework/Versions/A/Breakpad"
-    end
-
```

Comment:
Someone upstream here fixed the underlying issue in the release before the most recent.